XCUITests/FxScreenGraph.swift

```diff
@@ -718,9 +718,10 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
     }
 
     func type(text: String) {
-        text.forEach { char in
-            app.keys["\(char)"].tap()
-        }
+        app.typeText(text)
+//        text.forEach { char in
```

Comment:
If this method is changed as it is here, there is one Authentication test failing `testPromptPassCodeUponReentry`. If possible and if there is not  any strong reason for it, could it be as it is currently?

Reply:
Well, actually all Authentication tests fail. I have run the schema on BB, you can see the results [here](https://dashboard.buddybuild.com/apps/57bf25c0f096bc01001e21e0/build/5d03793e5c630e0001c3ca86#tests)